src/test/resources/application-integration.properties

```diff
@@ -0,0 +1 @@
+spring.cache.type=NONE
```

Comment:
can this just be in an application properties in the test resources folder?

Reply:
You then have to copy the whole of the application.properties file again as it overwrites. I've chosen this route so we don't have to copy everything. Happy for you to disagree though...

Reply:
😢 👍